tests/requirements.txt

```
@@ -1,9 +1,13 @@
+urllib3<2.0.0;python_version<="3.7"
+urllib3
 flake8
 pytest==4.6.9;python_version<="3.0"
 pytest>=6.0.0;python_version>="3.0"
 pytest-timeout
 requests-mock
 trivup>=0.8.3
+fastavro<1.8.0;python_version=="3.7"
```

question, not a comment: below it seems we support cp36, but here, no dependency will be installed for python 3.6, only 3.7 above, is this the intended behaviour?

---

does it work if you set ?
```suggestion
fastavro<1.8.0;python_version<="3.7"
```

---

Because there is just "fastavro" in the line below, it falls back to that and handles 3.6, so it's not a problem. I discussed with Pranav on a call earlier